### PR TITLE
Libreoffice 5.0beta1

### PR DESCRIPTION
**Pull request by yurikoles**

### CODE REVIEW

**Comment by fanquake** [comment]
Can you please squash your commits

File: Casks/libreoffice-beta.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'libreoffice-beta' do
+  version '5.0.0.0.beta1'
+  sha256 'f932b8d3fc8bf86a6aba5363982d3d14197f176325b2ffe4ce69bea5938288b7'
+  url "https://download.documentfoundation.org/libreoffice/testing/#{version.split('.0.beta')[0]}/mac/x86_64/LibreOfficeDev_#{version}_MacOS_x86-64.dmg"
```
Comment (vitorgalvao) [inline Casks/libreoffice-beta.rb]: There should be an empty line before this.

File: Casks/libreoffice-beta.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  gpg "#{url}.asc",
+      :key_id => 'c2839ecad9408fbe9531c3e9f434a1efafeeaea3'
+
```
Comment (vitorgalvao) [inline Casks/libreoffice-beta.rb]: And none here.

**Comment by yurikoles** [comment]
And now what?

**Comment by fanquake** [comment]
@yurikoles You just need to squash your commits.

**Pull request closed without merging**